Buildings/Controls/OBC/CDL/Logical/VariablePulse.mo

```diff
@@ -0,0 +1,329 @@
+within Buildings.Controls.OBC.CDL.Logical;
+block VariablePulse
+  "Generate boolean pulse with the width specified by input"
+
+  parameter Real samplePeriod(unit="s")
+    "Sample period of the width input";
+  parameter Real period(unit="s")
+    "Time for one pulse period";
+  parameter Real chaWidThr=0.05
+    "Threshold for checking if the width input is changed. It is the ratio of the value change to the original value"
+    annotation (Dialog(tab="Advanced"));
+  parameter Real zerWidThr=0.01
+    "Threshold for checking if the width input is greater than zero"
+    annotation (Dialog(tab="Advanced"));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uWid(
```

Comment:
Better just call this `u` if you agree it can't be anything else then the width.

Reply:
changed to `u`.